cmd/minishift/cmd/addon/addon_enable.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addon
+
+import (
+	"fmt"
+	"github.com/minishift/minishift/pkg/util/os/atexit"
+	"github.com/spf13/cobra"
+)
+
+const (
+	priorityFlag   = "priority"
+	addOnConfigKey = "addons"
+)
+
+var priority int
+
+var addonsEnableCmd = &cobra.Command{
+	Use:   "enable ADDON_NAME",
+	Short: "Enables the specified addon.",
+	Long:  "Enables the specified addon to be run after cluster creation",
+	Run:   runEnableAddon,
+}
+
+func init() {
+	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of this addon during addon execution.")
+	AddonsCmd.AddCommand(addonsEnableCmd)
+}
+
+func runEnableAddon(cmd *cobra.Command, args []string) {
+	addonName := args[0]
```

Comment:
I am seeing http://pastebin.centos.org/75476/ and I think it need to be fixed here.